addon/templates/components/sl-modal.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="modal-dialog">
+    <div class="modal-content">
+        {{yield aria-labelledby}}
```

Comment:
Based on in-person conversation, confirm whether this needs to be yielded or not.